src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -154,6 +186,68 @@ public IObservable<Unit> Push(ILocalRepositoryModel repository)
             });
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2201", Justification = "Prototype")]
+        public IObservable<Unit> SyncSubmodules(ILocalRepositoryModel repository)
+        {
+            return Observable.Defer(async () =>
+            {
+                var output = new StringWriter(CultureInfo.InvariantCulture);
+                var exitCode = await SyncSubmodules(repository.LocalPath, line => output.WriteLine(line));
+                if (exitCode != 0)
+                {
+                    // Replace with friendly message if Git.exe isn't on path.
+                    // If culture isn't English, user will see the local equivalent of:
+                    // "'git' is not recognized as an internal or external command"                    
+                    var message = output.ToString();
+                    if (exitCode == 1 && message.StartsWith("'git' is not recognized as an internal or external command,", StringComparison.Ordinal))
```

Comment:
This strikes me as a bit of a hack. Might there be a way to check whether `git.exe` is on the path and display an error rather than parsing the (localizable) error message?

Reply:
I can change it to use WHERE to find out where `Git.exe` is on the path.